sources/packages/web/src/components/students/applicationTracker/Completed.vue

```diff
@@ -1,35 +1,92 @@
 <template>
+  <!-- Student appeal - waiting approval -->
+  <application-status-tracker-banner
+    label="Your requested change is in progress"
+    icon="fa:fas fa-exclamation-triangle"
+    icon-color="warning"
+    background-color="warning-bg"
+    content="StudentAid BC is reviewing your requested change. If your requested change is approved, your application will be re-evaluated with a new assessment below."
+    v-if="assessmentDetails.appealStatus === StudentAppealStatus.Pending"
```

Comment:
can we move the `v-if` to line 4 as we are following in other areas

Reply:
**"As we are following in others areas"** is not an accurate statement, IMO.
I copied some of those from the below file where both "ways" are present, I did not feel that it was a strong "pattern" but we can agree on this.
https://github.com/bcgov/SIMS/blob/main/sources/packages/web/src/components/students/applicationTracker/InProgress.vue